pyre/check.py

```
@@ -1792,6 +1794,172 @@ def default_binary(backend):
+    return Path(str(artefact) + ".inputs")
+
+
+def stamp_artefact_inputs(artefact):
+    """Record the inputs an artefact was just built from, beside it."""
+    fingerprint = build_inputs_fingerprint()
+    if fingerprint is None:
+        return
+    try:
+        artefact_fingerprint_path(artefact).write_text(fingerprint, encoding="utf-8")
+    except OSError:
+        pass
```

Ruff (0.16.1)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)